### PR TITLE
docs: add what is jina

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

File: docs/get-started/what-is-jina.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+It is the part of Jina AI ecosystem that is there to bring your search to production. It is designed to build your applications as a set 
+of independent microservices that can be scaled independently and that used together in a pipeline can solve any search related task. 
+
+Jina gives the power to the developers to build their business logic and helps the user by providing abstractions that will help to modularize, serve and scale your applications in the cloud.
+
+Jina heavily uses other components of Jina Ecosystem as [**DocArray**](https://docarray.jina.ai/) to serve as the main data structure and [**Jina Hub**](https://hub.jina.ai/)  to help share, containerize and reuse 
+components of the search pipelines.
+
+Jina is designed to provide a similar experience when developing your solutions locally and when deploying them int he cloud, helping the developer
```
Comment (jacobowitz) [inline docs/get-started/what-is-jina.md:13]: typo
```suggestion
Jina is designed to provide a similar experience when developing your solutions locally and when deploying them in the cloud, helping the developer 
```

**Review by jacobowitz — CHANGES_REQUESTED** [review]
just a typo

File: docs/get-started/what-is-jina.md [hunk 2]
```diff
@@ -1,3 +1,26 @@
+# What is Jina?
+
+Jina is a neural search framework written in Python that helps developers build neural search applications that can be easily scaled and deployed in the Cloud.
+
+It is the part of Jina AI ecosystem that is there to bring your search to production. It is designed to build your applications as a set 
+of independent microservices that can be scaled independently and that used together in a pipeline can solve any search related task.
```
Comment (cristianmtr) [inline docs/get-started/what-is-jina.md:6]:
```suggestion
of separate microservices that can be scaled independently, but can be combined to solve search-related tasks.
```

File: docs/get-started/what-is-jina.md [hunk 3]
```diff
@@ -1,3 +1,26 @@
+# What is Jina?
+
+Jina is a neural search framework written in Python that helps developers build neural search applications that can be easily scaled and deployed in the Cloud.
+
+It is the part of Jina AI ecosystem that is there to bring your search to production. It is designed to build your applications as a set 
+of independent microservices that can be scaled independently and that used together in a pipeline can solve any search related task. 
+
+Jina gives the power to the developers to build their business logic and helps the user by providing abstractions that will help to modularize, serve and scale your applications in the cloud.
```
Comment (cristianmtr) [inline docs/get-started/what-is-jina.md:8]:
```suggestion
Jina gives the power to the developers to build their business logic. It guides the user with abstractions that will help to modularize, serve and scale your applications in the cloud.
```

In general, break up longer sentences. Much easier to read and follow

File: docs/get-started/what-is-jina.md [hunk 4]
```diff
@@ -1,3 +1,26 @@
...
+
+Jina heavily uses other components of Jina Ecosystem as [**DocArray**](https://docarray.jina.ai/) to serve as the main data structure and [**Jina Hub**](https://hub.jina.ai/)  to help share, containerize and reuse
```
Comment (cristianmtr) [inline docs/get-started/what-is-jina.md:10]:
```suggestion
Jina heavily uses other components of Jina Ecosystem: [**DocArray**](https://docarray.jina.ai/) to serve as the main data structure; [**Jina Hub**](https://hub.jina.ai/) to share, containerize and reuse 
```

File: docs/get-started/what-is-jina.md [hunk 5]
```diff
@@ -1,3 +1,26 @@
...
+
+Jina heavily uses other components of Jina Ecosystem as [**DocArray**](https://docarray.jina.ai/) to serve as the main data structure and [**Jina Hub**](https://hub.jina.ai/)  to help share, containerize and reuse 
+components of the search pipelines.
+
+Jina is designed to provide a similar experience when developing your solutions locally and when deploying them int he cloud, helping the developer 
+in every phase of the development cycle.
+
+## Design 
+
+Jina is designed as a lean and easy to use framework, and there are only two main concept to learn:
```
Comment (cristianmtr) [inline docs/get-started/what-is-jina.md:18]:
```suggestion
Jina is designed as a lean and easy-to-use framework. There are only two main concepts to learn:
```

File: docs/get-started/what-is-jina.md [hunk 6]
```diff
@@ -1,3 +1,26 @@
...
+
+- {ref}`Executor <executor-cookbook>` is a self-contained component and performs a single task on Documents.
```
Comment (cristianmtr) [inline docs/get-started/what-is-jina.md:20]: Here it's weird that we refer to `Documents`, but don't link to them, or mention them as a main component. Maybe link to  `docarray`

Comment (JoanFM) [reply docs/get-started/what-is-jina.md]: Done

File: docs/index.md [hunk 7]
```diff
@@ -88,12 +88,11 @@ Learn to share and reuse Executors from the Jina community.
 :caption: Get started
 :hidden:
 
-
+get-started/what-is-jina
 get-started/neural-search
```
Comment (hanxiao) [inline docs/index.md:92]:
```suggestion
get-started/neural-search
get-started/what-is-jina
```

File: docs/get-started/what-is-jina.md [hunk 8]
```diff
@@ -1,3 +1,26 @@
...
+It is the part of Jina AI ecosystem that is there to bring your search to production. It is designed to build your applications as a set 
+of separate microservices that can be scaled independently, but can be combined to solve search-related tasks.
+
+Jina gives the power to the developers to build their business logic. It guides the user with abstractions that will help to modularize, serve and scale your applications in the cloud.
+
+Jina heavily uses other components of Jina Ecosystem: [**DocArray**](https://docarray.jina.ai/) to serve as the main data structure; [**Jina Hub**](https://hub.jina.ai/) to share, containerize and reuse 
+components of the search pipelines.
+
+Jina is designed to provide a similar experience when developing your solutions locally and when deploying them int he cloud, helping the developer 
+in every phase of the development cycle.
```
Comment (nan-wang) [inline docs/get-started/what-is-jina.md:14]:
```suggestion


Jina empowers the developers to focus their business logic. Jina guides the user with abstractions that will help to modularize, serve and scale your applications in the cloud.

Jina helps the developer in every phase of the development cycle. Jina is designed to provide a similar experience when developing your solutions locally and in the cloud. It builds your applications with a set of separate microservices that can be scaled independently. 

As part of Jina AI ecosystem, Jina brings your search to production. [**DocArray**](https://docarray.jina.ai/) to serve as the main data structure; [**Jina Hub**](https://hub.jina.ai/) to share, containerize and reuse components of the search pipelines.


```

**Review by nan-wang — COMMENTED** [review]
Try to find a storyline for the first paragraph

**Pull request merged**